pkg/database.go

```diff
@@ -27,9 +27,14 @@ func insertComponentData(db *sql.DB, jobID, commit, createdAt string, totalSucce
 
 // insertQuayData inserts a record of Quay image pulls into the database.
 func insertQuayData(db *sql.DB, datetime string, count int, kind string) error {
+	if datetime == "" || kind == "" || count < 0 {
```

Comment:
Why the datetime for inserting any data into a table is not current timestamp?

Reply:
The graphs are designed to group data by date rather than exact timestamps. Using the full timestamp is unnecessary for our use case since we focus solely on daily or monthly aggregations.